.github/workflows/tic.yml

```diff
@@ -18,15 +18,16 @@ jobs:
   all:
     runs-on: ${{ matrix.config.os }}
 
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    name: ${{ matrix.config.os }} - r-${{ matrix.config.r }} - mlr3-${{ matrix.config.mlr3 }}
 
     strategy:
       fail-fast: false
       matrix:
         config:
           # use a different tic template type if you do not want to build on all listed platforms
-          - { os: ubuntu-latest, r: "devel" }
-          - { os: ubuntu-latest, r: "release", pkgdown: true, latex: true }
+          - { os: ubuntu-latest, r: "devel", mlr3: "release"}
+          - { os: ubuntu-latest, r: "release", mlr3: "release", pkgdown: true, latex: true }
+          - { os: ubuntu-latest, r: "release", mlr3: "devel" }
```

Comment:
This works and is explicit, you could also just set one for the devel runner like `devel: "true"` and then use `ci_has_env("devel")` in `tic.R`.